### PR TITLE
04.Vagrant

### PR DESCRIPTION
**Pull request by Zhenyakuch**

### CODE REVIEW

File: Evgeniya_Pehota/04.Vagrant/04.Vagrant.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Vagrant.configure("2") do |config|
+  config.vm.define "debian" do |debian|
+    debian.vm.box = "debian/bullseye64"
+    debian.vm.box_version = "11.20210409.1"
+    debian.vm.provider "virtualbox" do |vb|
+        vb.gui = false
+        vb.memory = "512"
+    end
+    debian.vm.provision "shell", inline: <<-SHELL
+      sudo useradd -m updater
```
Comment (pluhin) [inline Evgeniya_Pehota/04.Vagrant/04.Vagrant.md:22]: Please remove sudo, it is not needed

File: Evgeniya_Pehota/04.Vagrant/04.Vagrant.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  config.vm.define "centos" do |centos|
+    centos.vm.box = "centos/stream8"
+    centos.vm.box_version = "20210210.0"
+    centos.vm.provider "virtualbox" do |vb|
+          vb.gui = false
+          vb.memory = "1024"
+    end
+    centos.vm.provision "shell", inline: <<-SHELL
+      sudo useradd updater -s /bin/bash
+      sudo echo "updater:vagrant" | chpasswd
```
Comment (pluhin) [inline Evgeniya_Pehota/04.Vagrant/04.Vagrant.md:44]: Please remove sudo

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Evgeniya_Pehota/04.Vagrant/04.Vagrant.md (hunk 2 above)
Comment (Zhenyakuch) [reply Evgeniya_Pehota/04.Vagrant/04.Vagrant.md:44]: fixed

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**